content/guides/getting-started/component-test-troubleshooting.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Component Troubleshooting
+---
+
+There are times when you will encounter errors or unexpected behavior with
```

Comment:
Considering adding a section like this at the top as a general introduction. I just pulled this from https://docs.cypress.io/guides/references/troubleshooting but we can pull this into a partial if using the same content makes sense.

Reply:
dig it.